lua/auto-hlsearch.lua

```diff
@@ -50,15 +50,15 @@ local function init(config)
     if is_plugin_disabled then return end
 
     -- need to schedule, since noh doesn't work with autocmd. :h noh
-    vim.schedule(function() vim.cmd(":noh") end)
+    vim.schedule(function() vim.o.hlsearch = false end)
```

Comment:
can you please remove the schedule and the comment, since with vim.o there is no need to schedule anymore

```suggestion
    vim.o.hlsearch = false
```